### PR TITLE
Adding metadata and data upload documentation pages.

### PR DESCRIPTION
**Pull request by emmetaobrien**
This PR includes the following files:

CONP_main_data_documentation.htm: One page summary of how CONP uses DATS.json files and DataLad/GitAnnex for non-technical users.

DATS_fields.xlsx: Spreadsheet containing details of recommended/required/optional fields in CONP DATS.json schema.   Adapted from Jennifer Tremblay-Mercier's spreadsheet at https://docs.google.com/spreadsheets/d/1paQuDqySmjYUsGut9mXbXEdMOuPoUVJg/edit#gid=1995869063 with fields implemented in https://github.com/CONP-PCNO/conp-dataset/pull/125 added.

datalad_dataset_addition_procedure.md: Technical document on how to install Git/GitAnnex/Datalad and use them to upload and download data from CONP.  This will be updated imminently for modifications made by @gi114 at @jbpoline's request, and probably again early in the New Year once we move from DataLad 0.11 to 0.12.

The first file contains links to the other two, and also to the DATS.json file for the 1000GenomesProject dataset as an example/template, and to Patrick Park's draft DATS creator at https://dats-creator.herokuapp.com/ 

Comments, corrections and edits welcome.

### CODE REVIEW

**Comment by emmetaobrien** [comment]
Updated dataset preparation procedure to include modifications made by @gi114.

File: CONP_main_data_documentation.htm [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+The CONP DATS.json schema includes the following required fields:
+
+- Dataset title and description
+- Creators of the dataset
+- DOI or other unique identifier
+- License under which the dataset is distributed
+- Keywords and specification of data type
+
+
+The schema also includes many other recommended and optional fields.  Full details can be found <a href="../DATS_fields.xlsx">here</a>.  The example DATS.json file for the 1000 Genomes Project <a href="https://github.com/emmetaobrien/1000GenomesProject/blob/master/DATS.json">here</a> can be used as a template.  An interface allowing users to fill in required fields online is under development <a href="https://dats-creator.herokuapp.com/">here</a>.
```
Comment (xlecours) [inline CONP_main_data_documentation.htm]: A quick mention of `dataset_schema.json` would be helpful for those who are looking for structure guidance for properties.

File: CONP_main_data_documentation.htm [hunk 2]
```diff
@@ -0,0 +1,27 @@
+
+
+<h1>Uploading data for CONP</h1>
+
+<h2>1. Metadata</h2>
+
+We use the DATS.json file format to store metadata about CONP datasets. DATS.json is a flexible machine-readable structure allowing for sophisticated representation of a wide range of different types of datasets, from neuroimaging to genomics.  A DATS.json file must be prepared for each dataset.
+
+The CONP DATS.json schema includes the following required fields:
```
Comment (xlecours) [inline CONP_main_data_documentation.htm]: the excel file also has the identifier, types and version as required fields.

File: CONP_main_data_documentation.htm (hunk 1 above)
Comment (emmetaobrien) [reply CONP_main_data_documentation.htm]: Sure, that makes sense and I will do it now.

File: CONP_main_data_documentation.htm (hunk 2 above)
Comment (emmetaobrien) [reply CONP_main_data_documentation.htm]: Is the excel file now an authoritative final version ?

Comment (xlecours) [reply CONP_main_data_documentation.htm]: I see it the other way around. Necessity comes from usage. Those fields are nice to have but not providing them wont break anything. Not having a title could.

Comment (emmetaobrien) [reply CONP_main_data_documentation.htm]: Jen was talking yesterday about making a nice neat table in a Word document from that spreadsheet once JB signed off on the decisions Jen and I made updating the spreadsheet yesterday morning, and I am expecting to replace the excel file that is currently there, and also update the list in the main documentation page, when I receive that table.

**Comment by emmetaobrien** [comment]
I have just added Jen's table as a Word document and updated CONP_main_data_documentation.htm to point at it, so this should now be ready to go.

**Pull request merged**